cluster/manifests/skipper/deployment.yaml

```diff
@@ -94,6 +94,9 @@ spec:
         args:
           - "run.sh"
           - "skipper"
+{{ if eq .ConfigItems.skipper_validate_query "false" }}
+          - "-validate-query=false"
```

Comment:
Could be just `- "-validate-query={{ .ConfigItems.skipper_validate_query }}"` I guess

Reply:
yes

Reply:
fixed